src/components/Welcome/index.jsx

```diff
@@ -32,17 +32,21 @@ const Welcome = ({isModalOpen, getNewPresentation}) => {
     return (
         <div>
             <Dialog open={isModalOpen} disableScrollLock={true} style={{ padding: '0px 0px 0px 0px' }}>
-                <DialogTitle id="alert-dialog-title">{t(`welcome.editor`)}!</DialogTitle>
-                <DialogContent>{t(`presentation.create.new`)}</DialogContent>
+                {/* <DialogTitle id="alert-dialog-title">{t(`welcome.editor`)}!</DialogTitle>
```

Comment:
Can we uncomment this stuff? I had it in the test rebase branch, we shouldn’t need to have the translation stuff commented out anymore